src/Exscript/protocols/Protocol.py

```diff
@@ -561,10 +564,12 @@ def connect(self, hostname = None, port = None):
         @param hostname: The remote host or IP address.
         @type  port: int
         @param port: The remote TCP port number.
+        @type  init_timeout: int
```

Comment:
You don't expose that as a parameter, but you document it. I would remove the docstrings (and don't expose it).